pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -40,6 +43,18 @@ const (
 	referenceCloser   = ')'
 )
 
+var kedaNamespace, _ = util.GetClusterObjectNamespace()
+
+// getWatchNamespace returns the namespace the operator should be watching for changes
```

Comment:
This comment is a bit confusing the func doesn't return any string or similar, it's a bool

Reply:
Sorry for the typo, corrected it.